src/main/java/com/KacFlor/ShopSpring/controller/OrderController.java

```diff
@@ -39,6 +40,13 @@ public ResponseEntity<Optional<Order>> getById(@PathVariable("id") Integer id){
         return ResponseEntity.ok(order);
     }
 
+    @PreAuthorize("hasAnyAuthority('" + Role.Fields.USER + "', '" + Role.Fields.ADMIN + "')")
+    @GetMapping("/{id}/order-items")
+    public List<OrderItem> getAllOrderItemsByOrderId(@PathVariable("id") Integer id){
+        return this.orderService.getAllOrderItems(id);
```

Comment:
getAllByOrderItemsId?

Reply:
these are all order items by order ID

Reply:
ok then change the name to getAllByOrderId

Reply:
The names of the methods need to explain exacly what the method does, remember about it and fix the method names everywhere where it is needed. https://medium.com/swlh/how-to-better-name-your-functions-and-variables-e962a4ef335b